components/main-chef-wrapper/cmd/supermarket.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
```

Comment:
can replace copyright header with the one used in other files with year updated.